packages/run/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@rollup/plugin-run",
+  "version": "1.1.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "Run your bundle after you've built it",
+  "license": "MIT",
+  "repository": "rollup/plugins",
+  "author": "Rich Harris",
+  "homepage": "https://github.com/rollup/plugins/packages/run/#readme",
+  "bugs": "https://github.com/rollup/plugins/issues",
+  "main": "index.js",
+  "scripts": {
```

Comment:
missing `files` property

Reply:
@shellscape  Is the `files` property absolutely necessary? Based on the [npm package.json spec](https://docs.npmjs.com/files/package.json#files), certain files are always included, regardless of settings:

```
package.json
README
CHANGES / CHANGELOG / HISTORY
LICENSE / LICENCE
NOTICE
The file in the “main” field
```